primitives/src/pcs/mod.rs

```diff
@@ -92,13 +92,14 @@ pub trait PolynomialCommitmentScheme<E: PairingEngine> {
     /// &Self::ProverParam, ..)` or `commit(prover_param:
     /// Box<Self::ProverParam>, ..)` or `commit(prover_param:
     /// Arc<Self::ProverParam>, ..)` etc.
+    /// Also, the commitment is not necessarily hiding.
```

Comment:
Is is sometimes hiding and sometimes not, depending on the implementation? I suppose it's actually never hiding, in which case we should say it's explicitly **non hiding**.

Reply:
Hiding is usually required for many commitment scheme. However, in our use case, Plonk doesn't need PCS to be hiding. Also our KZG implementation is not hiding. I'm just putting this as some kind of warning.

Reply:
That's what I mean. Usually (always?) hiding would require an additional randomness input, and since in our API we don't even have it, it cannot be hiding, so we should just say it's **not hiding**.

Reply:
Even if there's no randomness input. The PCS could bake some rng inside to make it somewhat hiding. Isn't it?

Reply:
Second thought, I'll take your suggestion and propose a low-priority issue to add hiding property to the PCS trait.

Reply:
ok fair